### PR TITLE
[IMP] sale_order_event_attendee

### PR DESCRIPTION
**Pull request by leiremartinezdesantos**

### CODE REVIEW

File: sale_order_event_attendee/models/event.py [hunk 1]
```diff
@@ -1,20 +1,35 @@
...
+        elif self.order_status in ('done', 'sale'):
+            raise ValidationError(
+                _("You cannot cancel a participant with a "
+                  "confirmed Sale Order."))
+
     def action_confirm(self):
-        super(EventRegistration, self).action_confirm()
+        res = super(EventRegistration, self).action_confirm()
         if not self.partner_id:
             return
```
Comment (Daniel-CA) [inline sale_order_event_attendee/models/event.py]: No hay que devolver el resultado de action_confirm()?

**Comment by Daniel-CA** [comment]
Da error también el travis:

2021-07-26 07:40:20,619 5793 ERROR openerp_test odoo.addons.event_sale.tests.test_event_sale_ui: FAIL: TestUi.test_event_configurator

Traceback (most recent call last):

File "/home/travis/OCB-14.0/addons/event_sale/tests/test_event_sale_ui.py", line 29, in test_event_configurator

self.start_tour("/web", 'event_configurator_tour', login="admin")

File "/home/travis/OCB-14.0/odoo/tests/common.py", line 1435, in start_tour

res = self.browser_js(url_path=url_path, code=code, ready=ready, **kwargs)

File "/home/travis/OCB-14.0/odoo/tests/common.py", line 1419, in browser_js

self.fail('%s\n%s' % (message, error))

AssertionError: The test code "odoo.startTour('event_configurator_tour')" failed

Tour event_configurator_tour failed at step div[name="event_id"] input

**Review by alfredoavanzosc — APPROVED** [review]
:+1:

File: sale_order_event_attendee/models/sale_order.py [hunk 2]
```diff
@@ -10,6 +10,10 @@ def _calculate_order_line_qty(self):
         for line in self.order_line:
             if line.product_id.event_ok:
                 line_count = self.env['event.registration'].search_count(
-                    [('sale_order_line_id', '=', line._origin.id)]
+                    [('sale_order_id', '=', self.id),
+                     ('sale_order_line_id', '=', line._origin.id)]
                 )
-                line.product_uom_qty = line_count
+                if line_count == 0:
```
Comment (anajuaristi) [inline sale_order_event_attendee/models/sale_order.py]: Hay que eliminar el unlink.
Quitando eso, va perfecto.
El código a meter es: 
  ''' 
    if line_count > 0 
         line.product_uom_qty = line_count
   '''

**Review by anajuaristi — APPROVED** [review]

**Pull request merged**